utils/globals.js

```diff
@@ -0,0 +1,10 @@
+const APIS = {
+    EVERYTHING: '/everything'
+}
+
+const API_KEY = "70905943afe0477ab21103fdbb396454"
+
+export {
+    APIS,
+    API_KEY
+}
```

Comment:
I believe these should get declared in env file

Reply:
Fixed